src/util/logging.cpp

```diff
@@ -153,15 +153,18 @@ void Logging::initialize(
         const QDir& logDir,
```

Comment:
I think this should become a QString() because it is called with a QString() anyway. This way we skip the QDir() constructor if not needed, which is no no-op.  

Not sure what this means: 
https://github.com/qt/qtbase/blob/c34c6af3740052617ee9f0876e2720046f0e6bb7/src/corelib/io/qdir.cpp#L369

Reply:
Ping

Reply:
Is the `QDir()` constructor computationally expensive? Otherwise let's keep the proper semantic argument type. We need QDir later anyway.

Reply:
I am unsure about the line I have linked above. At least it is much more expensive than QString. 
I think QDir is more a helper class with file system access than a proper semantic for a directory string.

Reply:
Ok.